packages/calcite-components/src/components/tabs/tabs.tsx

```diff
@@ -133,6 +153,43 @@ export class Tabs {
    */
   @State() tabs: HTMLCalciteTabElement[] = [];
 
+  mutationObserver = createObserver("mutation", (mutationsList: MutationRecord[]) => {
+    console.log("mutationsList", mutationsList);
+    for (const mutation of mutationsList) {
+      const target = mutation.target as HTMLElement;
+      if (
+        target.nodeName === "CALCITE-TAB-NAV" ||
+        target.nodeName === "CALCITE-TAB-TITLE" ||
```

Comment:
FIXME: the current configuration of the mutation observer would only be invoked when direct children are added/removed. `tab-title`s are children of `tab-nav`s, so the observer would miss these.

Mutations from other children might be helping update `tab-titles`, so we could improve this in a follow-up PR for the current milestone.